src/backend/api/Fusion.Resources.Api/Middleware/ApiProblem.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System.Collections.Generic;
+using System.Net;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Serialization;
+
+namespace Fusion.Resources.Api.Middleware
+{
+    public class ApiProblem
```

Comment:
Dotnet core has a built-in class for this: 
https://docs.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.mvc.problemdetails?view=aspnetcore-3.1 
which adheres to IETF specification.

Can use helper method Problem(title: "Issues"....) to return errors in this format.

Perhaps using the standard dotnet core way of doing this will save us some work?